qa/L0_backend_python/unittest/test.sh

```diff
@@ -32,6 +32,7 @@ CLIENT_LOG="./client.log"
 EXPECTED_NUM_TESTS="1"
 TEST_RESULT_FILE='test_results.txt'
 SERVER_LOG="./inference_server.log"
+export CUDA_VISIBLE_DEVICES=0,1,2,3
```

Comment:
Will this work with all devices? Have you confirmed the L0_backend_python_jetson test would still pass?

Reply:
Pipeline: https://gitlab-master.nvidia.com/dl/dgx/tritonserver/-/pipelines/8146001
I've added changes for L0_backend_python to request 4 GPUs for testing, other tests set `CUDA_VISIBLE_DEVICES=0` , for this test I make visible all 4 gpus, and then in the end of the test change to  `CUDA_VISIBLE_DEVICES=0`. L0_backend_python--base green in the pipeline

Reply:
verifying on Jetson

Reply:
This test is excluded from jetson since GPU tensors are not supported on jetson devices.